### ISSUE TITLE
Look into chaching options of getJSON and getCSV

### ISSUE THREAD

**Issue by thombruce** [hugomodo/hugomodo-docs#1]
Currently loading the README for each theme and extension from GitHub API endpoint. Using GetJSON to do so.

The build for GitHub Pages by TravisCI appears to remain up to date (no cache), whereas my offline builds are not updating... I expect this means that the cache is, maybe, a comparison of page lastmod date to public file's last edited date. Unsure. Consider experiment: delete public folder and rebuild.

There is some chatter about cache refresh that started here: https://discourse.gohugo.io/t/configurable-cache-ttl-for-remote-content/10220/8

From which came this issue: https://github.com/gohugoio/hugo/issues/5404 And this PR: https://github.com/gohugoio/hugo/pull/5421

Looks like it was merged into v0.52.

Currently waiting on 0.54 for other issues on other site thombruce.gitlab.io where I'm relying on 0.47.
